portfolio/src/main/webapp/script.js

```diff
@@ -43,3 +28,44 @@ setInterval(() => {
 function scrollToProjects() {
   document.getElementById("projects").scrollIntoView({ behavior: "smooth" });
 }
+
+// Recursive function that fades out the tw-text div gradually over one second.
+// Takes an initial opacity value to start out.
+function fadeOutText(currOpacity) {
+  setTimeout(() => {
+    document.getElementById("tw-text").style.opacity = currOpacity;
+    if (currOpacity > 0) {
+      fadeOutText(currOpacity - 0.1);
+    }
+  }, 100);
+}
+
+// Typewriter effect loop that applies the effect to each string in the facts
+// array. Takes the current char index being written and the index of the current fact
+// in the facts array.
+function typeWriterEffect(charIndex, currentFactIndex) {
```

Comment:
You can reference to google js style guide to document each parameter:
https://google.github.io/styleguide/jsguide.html#jsdoc
It is easier to read than a general comment.

Reply:
Just committed a change that aligns my comments with the style guide.